From 2dbd80ffc4e8c769423e904842983bf70f73b0ac Mon Sep 17 00:00:00 2001
From: Hui Xia <hui.xia@intel.com>
Date: Wed, 22 Aug 2018 13:22:11 -0400
Subject: [PATCH 1601/1667] media: omap3isp: fix cross build issue

The build issue is found in arm-allmodconfig build and
casued by media_graph_walk_start definition change.
Modify driver to align with the definition change.

Signed-off-by: Hui Xia <hui.xia@intel.com>
---
 drivers/media/platform/omap3isp/ispvideo.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/omap3isp/ispvideo.c b/drivers/media/platform/omap3isp/ispvideo.c
index 218e6d7..c70efce 100644
--- a/drivers/media/platform/omap3isp/ispvideo.c
+++ b/drivers/media/platform/omap3isp/ispvideo.c
@@ -238,7 +238,7 @@ static int isp_video_get_graph_data(struct isp_video *video,
 		return ret;
 	}
 
-	media_graph_walk_start(&graph, entity);
+	media_graph_walk_start(&graph, &entity->pads[0]);
 
 	while ((entity = media_graph_walk_next(&graph))) {
 		struct isp_video *__video;
-- 
2.7.4

